app/view_objects/assessor_interface/further_information_request_view_object.rb

```diff
@@ -45,13 +46,37 @@ def review_items
               I18n.t(
                 "assessor_interface.further_information_requests.edit.check_your_answers",
               ),
-            fields: {
-              item.id => {
-                title: item_text(item),
-                value: item.text? ? item.response : item.document,
-              },
-            },
+            fields: review_items_fields(item),
           },
+          rows:
```

Comment:
```suggestion
          work_history_summary_list_rows:
```

`rows` on its own isn't very descriptive, this makes it clear what these rows are, specifically they're suitable for the summary list component and they're information about the work history.